alcs-frontend/src/app/features/board/board.component.ts

```diff
@@ -143,7 +143,7 @@ export class BoardComponent implements OnInit {
       status: application.status,
       title: `${application.fileNumber} (${application.applicant})`,
       assigneeInitials: application.assignee
-        ? `${application.assignee?.givenName.charAt(0)}${application.assignee?.familyName.charAt(0)}`
+        ? `${application.assignee?.givenName?.charAt(0)}${application.assignee?.familyName?.charAt(0)}`
```

Comment:
If I'm not mistaken assignee is of type UserDto, and UserDto has the initials set in auto mapper. I think we should be able to use that here

Reply:
You are very correct! I added initials to the front end DTO and it works great.